nextflow_schema.json

```
@@ -104,6 +104,20 @@
                 }
             }
         },
+        "downstream_csv_options": {
+            "title": "Downstream CSV options",
+            "type": "object",
+            "description": "Options to customize downstream csv creation",
+            "default": "",
+            "properties": {
+                "strandedness": {
+                    "type": "string",
+                    "default": "auto",
+                    "description": "Specifies the strandedness of RNA-Seq data for downstream sample sheet generation. This parameter does not affect the demultiplexing process but is used to generate the appropriate field in the nf-core/rnaseq samplesheet. Accepted values are 'unstranded', 'auto', 'forward', or 'reverse'.",
```

```suggestion
                    "description": "Specifies the strandedness of RNA-Seq data for downstream sample sheet generation. This parameter does not affect the demultiplexing process but is used to generate the appropriate field in the nf-core/rnaseq samplesheet,
```
make it an `enum` instead with this values